wls-gui-wahllokalsystem/src/components/common/dialogs/BaseDialog.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <v-dialog
+    :model-value="modelValue"
+    persistent
+  >
```

Comment:
Die maximale Breite entsprechend der Breite in der alten Anwendung

```suggestion
    persistent
    max-width="648px"
  >
```

Reply:
[58207db](https://github.com/it-at-m/Wahllokalsystem/pull/1430/commits/58207dbf9cb7884bc67746e4fb6e9d027690e17c)